### PR TITLE
updated with phoenix 1.4 specific changes

### PR DESCRIPTION
**Pull request by cultofmetatron**
I got subscriptions working in phoenix 1.4 and added here some updates that would need to be followed in order to get working

### CODE REVIEW

File: guides/subscriptions.md [hunk 1]
```diff
@@ -37,6 +37,25 @@ line:
...
+      # Start the Ecto repository
+      MyAppWeb.Repo,
+      # Start the endpoint when the application starts
+      MyAppWeb.Endpoint,
+      {Absinthe.Subscription, [MyAppWeb.Endpoint]}
+    ]
+
+    # See https://hexdocs.pm/elixir/Supervisor.html
+    # for other strategies and supported options
+    opts = [strategy: :one_for_one, name: Blinq.Supervisor]
```
Comment (maartenvanvliet) [inline guides/subscriptions.md]: What is the `Blinq.Supervisor`? I fear it may confuse readers of the guide

Comment (bruce) [reply guides/subscriptions.md]: Agreed. Maybe `MyApp`, as we use frequently?

**Comment by bruce** [comment]
Going to hook @benwilson512 in as well as a reviewer.

File: guides/subscriptions.md (hunk 1 above)
Comment (cultofmetatron) [reply guides/subscriptions.md]: I made the change as requested. that was an accidental copy over from the POC I put together

**Comment by bruce** [comment]
I made some minor adjustments to wording. I'm going to go ahead and get this in since it's a doc-only change.

**Pull request merged**